pypit/arsciexp.py

```diff
@@ -643,9 +647,14 @@ def MasterWave(self, fitsdict, sc, det):
             if settings.argflag["reduce"]["calibrate"]["wavelength"] == "pixel":
                 mswave = self._tilts[det - 1] * (self._tilts[det - 1].shape[0]-1.0)
             else:
-                wv_calib = self._wvcalib[det - 1]
-                mswave = arutils.func_val(wv_calib['fitc'], self._tilts[det - 1], wv_calib['function'],
+                ok_slits = np.where(~self._maskslits[det-1])[0]
```

Comment:
I've gotten some weird behavior with `~` before.  It was long enough ago that I don't actually remember what the behavior was, and it could easily have been me doing something wrong.  But ever since I've preferred to use `np.invert` instead.

Reply:
I've had issues using ```not``` but I think ```~``` is ok.  
And I find it much easier to read than ```np.invert```

Am going to keep it for now but won't fuss if someone else
makes that change.  ;)